File: go/wam/4838.go
const WamUpdatesTabSearch = 4838 // channel: regular

type UpdatesTabSearchEvent struct {
	ChannelsFollowedCount int64 `wam:"channels_followed_count"` // field 1
	RecentStatusItemCount int64 `wam:"recent_status_item_count"` // field 2
	RecentStatusRowCount int64 `wam:"recent_status_row_count"` // field 3
	UpdateTabSearchEventType UPDATETABSEARCHEVENTTYPE `wam:"update_tab_search_event_type"` // field 4
	ChannelsAdminCount int64 `wam:"channels_admin_count"` // field 5
	ViewedStatusItemCount int64 `wam:"viewed_status_item_count"` // field 6
	ViewedStatusRowCount int64 `wam:"viewed_status_row_count"` // field 7
	PremiumChannelsFollowedCount int64 `wam:"premium_channels_followed_count"` // field 8
	UpdatesTabSearchResultType UPDATESTABSEARCHRESULTTYPE `wam:"updates_tab_search_result_type"` // field 9
	UpdatesTabSearchSessionId string `wam:"updates_tab_search_session_id"` // field 10
	UpdatesTabSearchModeType UPDATESTABSEARCHMODETYPE `wam:"updates_tab_search_mode_type"` // field 11
	UnifiedSessionId string `wam:"unified_session_id"` // field 12
	UpdatesTabSessionId int64 `wam:"updates_tab_session_id"` // field 13
}

File: ts/wam/4838.ts
export const WamUpdatesTabSearch = 4838 as const

export interface UpdatesTabSearchEvent {
  /** field 1, wire `channels_followed_count` */
  channelsFollowedCount?: number
  /** field 2, wire `recent_status_item_count` */
  recentStatusItemCount?: number
  /** field 3, wire `recent_status_row_count` */
  recentStatusRowCount?: number
  /** field 4, wire `update_tab_search_event_type` */
  updateTabSearchEventType?: UPDATETABSEARCHEVENTTYPE
  /** field 5, wire `channels_admin_count` */
  channelsAdminCount?: number
  /** field 6, wire `viewed_status_item_count` */
  viewedStatusItemCount?: number
  /** field 7, wire `viewed_status_row_count` */
  viewedStatusRowCount?: number
  /** field 8, wire `premium_channels_followed_count` */
  premiumChannelsFollowedCount?: number
  /** field 9, wire `updates_tab_search_result_type` */
  updatesTabSearchResultType?: UPDATESTABSEARCHRESULTTYPE
  /** field 10, wire `updates_tab_search_session_id` */
  updatesTabSearchSessionId?: string
  /** field 11, wire `updates_tab_search_mode_type` */
  updatesTabSearchModeType?: UPDATESTABSEARCHMODETYPE
  /** field 12, wire `unified_session_id` */
  unifiedSessionId?: string
  /** field 13, wire `updates_tab_session_id` */
  updatesTabSessionId?: number
}